docs/changelog/97282.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 97282
```

Comment:
Should we keep one changelog?
I'm not sure whether we should keep this or the other.
Maybe because this backport is a while after the original change, let's keep this one?